bedrock/mozorg/templates/mozorg/contribute/studentambassadors/join.html

```diff
@@ -29,7 +29,7 @@
 
 {% block billboardcontent %}
   <form method="POST" class="billboard {% if form.non_field_errors() or form.errors %}has-errors{% endif %}"
-        id="ambassadors-form" action="{{ secure_url() }}#ambassadors-form">
+        id="ambassadors-form" action="{{ request.build_absolute_uri(request.path) }}#ambassadors-form">
```

Comment:
Any reason this couldn't use the `url()` helper? e.g. `url('mozorg.contribute.studentambassadors.join')`